index.html

```diff
@@ -134,7 +134,7 @@ <h3>Meshes, Materials, Import</h3>
           <ul>
             <li><a href="docs/meshes-materials-import/15-blender-tips-for-Horizon-assets/">15 Blender tips for Horizon
                 assets</a></li>
-            <li><a href="docs/meshes-materials-import/3d-modeling-in-Worlds-101/">3D modeling in Worlds 101</a></li>
+            <li><a href="docs/meshes-materials-import/3D-modeling-in-Worlds-101/">3D modeling in Worlds 101</a></li>
```

Comment:
I presume the jekyll transformer is now preserving case? Was it not before? Or was it the old 'client side rendering' code that was doing the lower casing?